lib/oauth2/client.rb

```diff
@@ -26,6 +26,8 @@ class Client # rubocop:disable Metrics/ClassLength
     # @option opts [Boolean] :raise_errors (true) whether or not to raise an OAuth2::Error
     # @options opts [Logger] :logger (::Logger.new($stdout)) which logger to use when OAUTH_DEBUG is enabled
     #  on responses with 400+ status codes
+    # @options opts [String] :dig_nested_keys_for_access_token comma-separated list of attributes to dig within the response hash to
+    #  retrieve the Oauth access token. e.g. for Pinterest API v3: "data,access_token"
```

Comment:
Is there a reasons to make this a comma delimited string vs just an array of strings?